openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/ORSGraphHopper.java

```diff
@@ -361,6 +361,29 @@ else if (ALT_ROUTE.equalsIgnoreCase(algoStr))
 				if (weighting instanceof TurnWeighting)
 	                ((TurnWeighting)weighting).setInORS(true);
 
+				weighting = createTimeDependentAccessWeighting(weighting, tMode, algoStr);
```

Comment:
Is it correct to enforce the time dependent weighting to be an access weighting here? should it not be something more general?

Reply:
This call is supposed to wrap any super `weighting` (say `shortest`, `fastest`, or `td_fastest` into a `TimeDependentAccessWeighting`  similarly as `createTurnWeighting`, so I don't really see an issue here.

Reply:
I understand that, I was wondering whether this would be correct under any circumstance, or whether a generic td weighting would be better. This generic weighting can then be made an accessWeighting if the conditions (I do not know whats necessary for that) are met

Reply:
So as long as time-dependent routing is enabled and a conditional access storage is available to a given encoder it makes sense to have `TimeDependentAccessWeighting` enabled for the sake of correctness of routes. Do you have any particular time-dependent weighting in mind which would fit into this scheme? When it comes to the fastest-based weightings: it doesn't seem feasible to me to have them implemented as wrapper weightings because of the conversion from speed to the time domain which happens in them internally.

Reply:
I do not have one in mind. But we might come to that point later. Is the AccessStorage always available? Are there circumstances where it is not created?

Reply:
There is currently no mechanism implemented which would prevent it from being created.

Reply:
OK, seems that it would be reasonable to add to `createTimeDependentAccessWeighting` method a check for the presence of `conditional_access` encoded value for a given flag encoder. I will fix this.

Reply:
Done!